ee.ria.qdigidoc4.yml

```
@@ -0,0 +1,198 @@
+
+  - name: qdigidoc4
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/open-eid/DigiDoc4-Client/releases/download/v4.5.0/qdigidoc4_4.5.0.4451-2004.tar.xz
+        sha256: ff44f3cadd2d427d0eecba8f8dbc1dd96dceab34ea5237b9c82bfb36f98ad772
+      - type: patch
+        path: qdigidoc4/QPKCS11.patch
+      - type: patch
+        path: qdigidoc4/client-CMakeLists.patch
+      - type: file
+        path: ee.ria.qdigidoc4.metainfo.xml
+      # See guide for building in a sandboxed environment:
+      # https://github.com/open-eid/DigiDoc4-Client/wiki/DeveloperTips#building-in-sandboxed-environment
+      - type: file
+        url: https://ec.europa.eu/tools/lotl/eu-lotl.xml
+        sha256: ccb08c603caea3e5501d30acea41ff5253fbf8f304afea6edaa4b3d537480541
+        dest: client
+        dest-filename: eu-lotl.xml
+      - type: file
+        url: https://sr.riik.ee/tsl/estonian-tsl.xml
+        sha256: 472d248b30c849873173c6dee0a3de417996110c2704cf5a261bca99d3c0d1d4
+        dest: client
+        dest-filename: EE.xml
+      - type: file
+        path: qdigidoc4/TSL.qrc
+        dest: client
+        dest-filename: TSL.qrc
+      - type: file
+        url: https://id.eesti.ee/config.json
+        sha256: 7b3b2af587571d29df3fde21f8d6f86bb6afce0d0325487aedd3fc799d3f65de
+        dest: common
+        dest-filename: config.json
+      - type: file
+        url: https://id.eesti.ee/config.rsa
+        sha256: 4b435b12f142ec41eca87c23c17a868706c38badba573812d96e5dc8230ba714
+        dest: common
+        dest-filename: config.rsa
+      - type: file
+        url: https://id.eesti.ee/config.pub
+        sha256: 29e05778ce98b5197963266bdc292af80a9406cccc3fe5ef7ca2b75b1beb34eb
+        dest: common
+        dest-filename: config.pub
+      - type: patch
+        path: qdigidoc4/common-CMakeLists.patch
+      # qdigidoc4 saves files in $TMPDIR before opening them. Since this is inaccessible, we provide a wrapper script,
+      # as advised here: https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access
+      - type: script
+        dest-filename: run.sh
+        commands:
+          - env TMPDIR="$XDG_RUNTIME_DIR/app/$FLATPAK_ID" qdigidoc4 "$@"
+    post-install:
+      - install run.sh "${FLATPAK_DEST}/bin/run.sh"
+      - install -Dm644 ee.ria.qdigidoc4.metainfo.xml $FLATPAK_DEST/share/metainfo/ee.ria.qdigidoc4.metainfo.xml
+      - ln -s $FLATPAK_DEST/share/locale/et/LC_MESSAGES/nautilus-qdigidoc.mo $FLATPAK_DEST/share/locale/et/nautilus-qdigidoc.mo
+      - ln -s $FLATPAK_DEST/share/locale/ru/LC_MESSAGES/nautilus-qdigidoc.mo $FLATPAK_DEST/share/locale/ru/nautilus-qdigidoc.mo
+      - mv $FLATPAK_DEST/share/applications/qdigidoc4.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} --set-key=Exec --set-value="run.sh" /app/share/applications/${FLATPAK_ID}.desktop
```

```suggestion
      - desktop-file-edit --set-key=Exec --set-value="run.sh" /app/share/applications/${FLATPAK_ID}.desktop
```

`rename-icon` does it for you.